library/Core/src/Juvix/Core/Erasure/Algorithm.hs

```diff
@@ -79,10 +78,10 @@ eraseFunction ::
   IR.Function primTy1 primVal1 ->
   m (Erasure.Function primTy2 primVal2)
 eraseFunction (IR.Function name usage ty clauses) = do
-  let (tys, ret) = piTypeToList (IR.quote0 ty)
-  clauses <- flip mapM clauses $ \(IR.FunClause _tel patts term _rhsTy _catchAll _unreachable) -> do
-    let ty_ret = listToPiType (drop (length patts) tys, ret)
-    (patts, ty) <- erasePatterns (patts, (tys, ret))
+  let (tys, ret) = piTypeToList (IR.quote ty)
+  clauses <- flip mapM clauses $ \(IR.FunClause _tel patts _term _rhsTy _catchAll _unreachable) -> do
+    -- let ty_ret = listToPiType (drop (length patts) tys, ret)
```

Comment:
this is not used, and is the only thing that calls `piTypeToList`. @andy-morris can we delete these lines?